Resources/Prototypes/SimpleStation14/Entities/Objects/Magic/Medical/asclepius.yml

```
@@ -0,0 +1,24 @@
+- type: entity
+  parent: BaseItem
+  id: StaffAsclepius
+  name: staff of Asclepius
+  description: A wooden rod about the size of your forearm with a snake carved around it, winding it's way up the sides of the rod. Something about it seems to inspire in you the responsibilty and duty to help others.
```

```suggestion
  description: A wooden rod with a snake carved around it, winding it's way up the sides of the rod. Something about it seems to inspire in you the responsibility and duty to help others.
```
Physical identifiers generally aren't great in item descriptions. Think Felinid forearm vs Oni forearm :P